cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
```

Comment:
```suggestion
    weak var delegate: PaymentViewControllerDelegate
```
Do not hold strong reference on delegate